app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -55,12 +58,55 @@ class MainActivity : ComponentActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background,
                 modifier = Modifier.fillMaxWidth()) {
-                    RecycleSearch("Android", products)
```

Comment:
I removed "Android" here as it's part of the beginning template and not necessary.  It can be replaced with something relevant to you project.

Reply:
Is the name parameter used at all?  If not, remove it entirely.